docs/tutorial/mlflow-kubeflow.rst

```diff
@@ -0,0 +1,185 @@
+Getting Started with Charmed MLflow and Kubeflow
+================================================
+
++-----------+---------+
+| Component | Version |
++-----------+---------+
+|   MLflow  |    2    |
++-----------+---------+
+| Kubeflow  | 1.7     |
++-----------+---------+
```

Comment:
This will be rendered correctly somehow, but let's please fix it:
```suggestion
+-----------+---------+
| Component | Version |
+-----------+---------+
|   MLflow  |    2    |
+-----------+---------+
|  Kubeflow |   1.7   |
+-----------+---------+
```